macostool.sh

```diff
@@ -0,0 +1,32 @@
+# This is a command to check weather your system is a Mac OS or not.
```

Comment:
I have taken out some commet section but need to install python and go in mac for the tools to get installed

Reply:
Yes thats fine will upload in the upcoming pull request